gateway/cpanfile

```diff
@@ -1,2 +1,2 @@
-requires 'Test::APIcast', '0.03';
+requires 'Test::APIcast', '0.05';
```

Comment:
I see that you released `0.06` some days ago. Any reason not to use that one?

Reply:
No reason. 0.06 was released after I changed this. Will update 👍